app/assets/javascripts/home/HomePage.js

```diff
@@ -37,6 +37,7 @@ const styles = {
   },
   column: {
     flex: 1,
+    width: 0,
```

Comment:
This fixes a related issue with making sure the two-column layout distributes extra space evenly and doesn't actually flex.